docs/api/workers.md

```
@@ -122,6 +122,65 @@ Messages are automatically enqueued until the worker is ready, so there is no ne
+
+**Bun (with fast paths):**
+
+```
+postMessage({ prop: 11 chars string, ...9 more props }) - 648ns
+postMessage({ prop: 14 KB string, ...9 more props })    - 719ns
+postMessage({ prop: 3 MB string, ...9 more props })     - 1.26µs
+```
+
+**Node.js v24.6.0 (for comparison):**
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)